code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm

```diff
@@ -228,7 +228,11 @@ Difficulty: Hard
 				addtimer(CALLBACK(src, .proc/devour, L), 2)
 	SLEEP_CHECK_DEATH(1, src)
 
-
+/mob/living/simple_animal/hostile/megafauna/bubblegum/devour(mob/living/yummy_food)
+	. = ..()
+	if(. == TRUE) // a corpse was devoured
+		// bubblegum bubblegum in a dish, how many corpses do you wish?
+		new /obj/item/food/bubblegum/bubblegum(src)
```

Comment:
You're spawning the bubblegum INSIDE the bubblegum mob, not on the floor where the body is killed.

Reply:
Sounds like a feature. Buuuuut I guess we do want the bubblegum to be eaten by _human_ mobs and not bubblegum himself...

Reply:
Fixed with a58570e8753253ce89eee3dfe1df10086aee27b5